portfolio/src/main/java/com/google/sps/servlets/Form2.java

```diff
@@ -0,0 +1,5 @@
+package com.google.sps.servlets;
+
+public class Form2 {
```

Comment:
"Form2" is not a good name considering what this class does. Try to think of a better name depending on what this file does :)

Reply:
Okay, I will but its kinda hard getting good names

Reply:
Actually Chen I name it form2 because its part of the feature that has a form to eat and it helps the form do what its supposed to do.

Reply:
A suggestion would be "NameForm" or "InputName" since this would tell that this class stores names. But I'll leave you to decide.